packetbeat/tests/system/test_0006_wsgi.py

```diff
@@ -89,8 +89,8 @@ def test_send_headers_options(self):
         assert len(objs) == 1
         o = objs[0]
 
-        assert "http.request_headers" not in o
-        assert "http.response_headers" not in o
+        assert "http.requestHeaders" not in o
```

Comment:
it looks like we are breaking BC here. For the output as json we always use lowercase naming with the underline.